www/src/data/strings/components.yaml

```diff
@@ -0,0 +1,14 @@
+docsTableOfContents:
+  title: Table of Contents
+markdownPageFooter:
+  text: Edit this page on GitHub
+prevAndNext:
+  prevLabel: Previous
+  nextLabel: Next
+searchForm:
```

Comment:
I didn't update search-form.js because it's a class component that's too detailed  and I didn't feel like making a render props version of `useLocalizedStrings` for this WIP.